src/codemagic/tools/xcode_project.py

```diff
@@ -214,6 +214,8 @@ def build_ipa(self,
         xcodebuild = self._get_xcodebuild(**locals())
         clean and self._clean(xcodebuild)
 
+        xcodebuild.show_build_settings()
```

Comment:
```suggestion
        self.logger.info(Colors.BLUE(f'Check {(xcodebuild.workspace or xcodebuild.xcode_project).name} build settings'))
        xcodebuild.show_build_settings()
```

Perhaps it makes sense to add human-readable log statement about what is going on. Without anything we have just 
```shell
Execute "xcodebuild -project /path/to/project.xcodeproj -scheme <scheme_name> -showBuildSettings"
```
in the very beginning, which does not look especially nice.